src/AcmProgressTracker/AcmProgressTracker.tsx

```diff
@@ -114,6 +125,22 @@ export function AcmProgressTracker(props: AcmProgressTrackerProps) {
                                     {step.statusSubtitle}
                                 </Text>
                             </TextContent>
+                            {step.link && (
+                                <AcmButton
+                                    className={classes.button}
+                                    variant="link"
+                                    icon={<ExternalLinkAltIcon size="sm" />}
+                                    iconPosition="right"
+                                    onClick={() => {
+                                        /* istanbul ignore next */
+                                        window.open(step.link?.linkUrl)
+                                    }}
+                                >
+                                    <Text className={classes.link} component="a">
```

Comment:
Why is the Text component needed here, and making it an anchor tag?

Reply:
Maybe you were trying to get the actually hyperlink behavior, if so, you just need to set the `AcmButton` prop to have a `isInline` prop and it should look like a regular hypertext link